staging/src/k8s.io/component-base/metrics/prometheus/clientgo/BUILD

```diff
@@ -0,0 +1,31 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
It's not a move (as in the commit message). It's a duplication.

Reply:
okay, i will amend the commit message.